internal/stanza/converter.go

```diff
@@ -220,20 +198,25 @@ func (c *Converter) workerLoop() {
 		case <-c.stopChan:
 			return
 
-		case e, ok := <-c.workerChan:
+		case entries, ok := <-c.workerChan:
 			if !ok {
 				return
 			}
 
-			lr := convert(e)
-			resourceID := getResourceID(e.Resource)
+			workerItems := make([]workerItem, 0, len(entries))
+
+			for _, e := range entries {
+				lr := convert(e)
+				resourceID := getResourceID(e.Resource)
+				workerItems = append(workerItems, workerItem{
+					Resource:   e.Resource,
+					ResourceID: resourceID,
+					LogRecord:  lr,
+				})
+			}
 
 			select {
-			case c.batchChan <- &workerItem{
-				Resource:   e.Resource,
-				ResourceID: resourceID,
-				LogRecord:  lr,
-			}:
+			case c.batchChan <- workerItems:
```

Comment:
We have a bit of an overloaded term here, where we're doing:

`batch (into a slice of entries) => convert => batch (by resource)`

Maybe the latter `batch` can be renamed for clarity?

Reply:
Yeah, I think "aggregate" is more clear for the latter case; Are we OK with that verbiage?

Reply:
Yeah, that's definitely an improvement.

Reply:
Replaced all references I could find of this action as "batching" with "aggregating". Also found that some of the docs had some wrong descriptions because of some of the changes with both the last PR and this one, so I fixed those as well.